modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -76,15 +108,7 @@ public void testInit() {
     public void testFinish() {
 
         RestAssured.basePath = StringUtils.EMPTY;
-    }
-
-    @DataProvider(name = "restAPIUserConfigProvider")
-    public static Object[][] restAPIUserConfigProvider() {
-
-        return new Object[][]{
-                {TestUserMode.SUPER_TENANT_ADMIN},
-                {TestUserMode.TENANT_ADMIN}
-        };
+        customIdPId = null;
```

Comment:
This does not delete the idp from the IS. Delete the idp here.

Reply:
This is fixed.